irods/test/library_features_test.py

```diff
@@ -0,0 +1,33 @@
+#! /usr/bin/env python
+from __future__ import absolute_import
+import os
+import sys
+import unittest
+
+import irods.test.helpers as helpers
+
+class TestLibraryFeatures(unittest.TestCase):
+
+    def setUp(self):
+        self.sess = helpers.make_session()
+
+    def tearDown(self):
+        """Close connections."""
+        self.sess.cleanup()
+
+    def test_library_features__issue_556(self):
+        if self.sess.server_version < (4, 3, 1):
+            self.skipTest('force flag unneeded for create in iRODS > 4.2.8')
```

Comment:
Guessing this message should be... something else.

On that note, perhaps we should include a test which *only* runs against iRODS servers older than 4.3.1 and asserts that the new `Exception` is `raise`d as expected.

Reply:
Good grief.  Yes, of course. It's going to be something else.

Reply:
> Guessing this message should be... something else.
> 
> On that note, perhaps we should include a test which _only_ runs against iRODS servers older than 4.3.1 and asserts that the new `Exception` is `raise`d as expected.
Took care of the first part.

Reply:
> On that note, perhaps we should include a test which _only_ runs against iRODS servers older than 4.3.1 and asserts that the new `Exception` is `raise`d as expected.

I can write this test, but it seems a little ... overkill maybe?  Will defer if consensus shows otherwise.

Reply:
> I can write such a test, but it seems a little ... overkill maybe? Will defer if consensus shows otherwise. Agree it helps for completeness, but since we're programming to the nicer things generally (iRODS >= 4.3.1) and I don't see the possibility of the version exception _not being thrown_ as a significant one.

That seems reasonable to me. No need for the suggested test.

Reply:
Ok , seems right.  Most everyone will be living in 4.3.3 land soon, right ; ) ?